pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -423,6 +424,7 @@ func buildKubeApiserverConfig(
 	genericConfig.Authorizer = kubeAuthorizer         // this is used to fulfill the kube SAR endpoints
 	genericConfig.AdmissionControl = admissionControl
 	genericConfig.RequestContextMapper = requestContextMapper
+	genericConfig.DisabledPostStartHooks.Insert(rbacrest.PostStartHookName)
```

Comment:
I would vote for turning this into a direct post-rebase follow-up as it is only cosmetic. The storage test runs fine now, but we have the bootstrap hook with its /healthz endpoint.

@liggitt are you ok with a follow-up?